python/pyhail/dataset.py

```diff
@@ -970,14 +1251,78 @@ def mendel_errors(self, output, fam):
     def pca(self, scores, loadings=None, eigenvalues=None, k=10, arrays=False):
         """Run Principal Component Analysis (PCA) on the matrix of genotypes.
 
+        **Examples**
+
+        ``pca`` expects a variant dataset with biallelic autosomal variants. Scores are computed and stored as sample annotations of type Struct by default; variant loadings and eigenvalues can optionally be computed and stored in variant and global annotations, respectively.
+
+        To compute the first 10 principal component scores and store as annotations of type Struct:
+
+        >>> vds1 = hc.read('example.vds').pca('sa.pca')
+
+        The annotation ``sa.pca`` on ``vds1`` is a Stuct with fields ``PC1`` to ``PC10`` of type Double, so for example ``sa.pca.PC1`` is the score from the top PC.
+
+        To compute the first 5 principal component scores, loadings, and eigenvalues, and store as annotations of type Array[Double]:
```

Comment:
I don't think you need to say the type here. It's documented in the annotations section.